src/utils/npm/npm-prefix.js

```diff
@@ -0,0 +1,16 @@
+const { prefix } = require("../prefix");
+const { useYarn } = require("./use-yarn");
+
+/**
+ * Returns the prefix to use with NPM.
+ * @param {string} linter - the name of the linter to use
+ * @param {string} dir - the directory to use to check the Yarn lockfile.
+ * @returns {string} - Prefix to use with NPM.
```

Comment:
For consistency, please capitalize the start of JSDoc descriptions and omit the full stop at the end.

Reply:
Will do!